### PR TITLE
Improve package description

### PR DESCRIPTION
**Pull request by fregante**
"Various" is selling this package short. Feel free to reword further

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"name": "eslint-plugin-unicorn",
 	"version": "42.0.0",
-	"description": "Various awesome ESLint rules",
+	"description": "More than 100 powerful ESLint rules",
```
Comment (fisker) [inline package.json:4]:
```suggestion
	"description": "World's mightiest ESLint rules",
```

Comment (fregante) [reply package.json:4]:
```suggestion
	"description": "ESLint rules that really whip the llama's ass",
```

Comment (fisker) [reply package.json:4]: You are an old man!

Comment (fregante) [reply package.json:4]: image

**Review by fisker — APPROVED** [review]

**Pull request merged**